go-magic-link/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"text/template"
+
+	"github.com/workos-inc/workos-go/pkg/passwordless"
+	"github.com/workos-inc/workos-go/pkg/sso"
+)
+
+type Profile struct {
+	Email   string
+	Session string
+}
+
+func main() {
+	address := ":8000"
+	apiKey := ""
+	clientID := ""
```

Comment:
Looks like client ID isn't necessary for this app, may want to remove that variable from line 23